Forwarded: No, but should be
Subject: Revert upstream "MDEV-24630: MY_RELAX_CPU assembly instruction upgrade"
Description:
 It was noticed that MariaDB 10.5.10 regressed and no longer built on armhf.
 For details see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=988629
 Reverting the upstream commit
 https://github.com/MariaDB/server/commit/76d2846a71a155ee2861fd52e6635e35490a9dd1
 is likely to fix it and make armhf build again.

--- include/my_cpu.h
+++ include/my_cpu.h
@@ -84,7 +84,7 @@ static inline void MY_RELAX_CPU(void)
   __ppc_get_timebase();
 #elif defined __GNUC__ && (defined __arm__ || defined __aarch64__)
   /* Mainly, prevent the compiler from optimizing away delay loops */
-  __asm__ __volatile__ ("isb":::"memory");
+  __asm__ __volatile__ ("":::"memory");
 #else
   int32 var, oldval = 0;
   my_atomic_cas32_strong_explicit(&var, &oldval, 1, MY_MEMORY_ORDER_RELAXED,
